### DISCUSSION TITLE
cannot download a new engine

### DISCUSSION THREAD

**Discussion by ebow77** [The-Wineskin-Project/WineskinServer#189] [Q&A]
Hi, I just installed Wineskin Winery 1.8.4.1 and I'm stuck at the next step: 

> Under "Installed Engines", you need to click [+] to access a new window.

When I press the + button, nothing happens. The only button that does something is "Download Packages Manually" which takes me to a mega.nz page that wants me to enter a decryption key. 

I'm on Mac OS 10.13.6 High Sierra. How can I get a list of engines to appear? 

edit: is the tool called Wineskin, or Winery? Other writeups online called it Wineskin Winery. I see the Winery folder, which is where I got 1.8.4.1 from. But just now I noticed the readme for the main wineskin folder includes a (not recommended) step of downloading Winery 1.8.4.2. It's not clear if they're different, or which I should be using. 

![image](https://user-images.githubusercontent.com/95896618/145512522-0f984fc9-a125-4235-9f04-0a63a8f18b78.png)

**Comment by Gcenx** [comment]
The readme recommends installing via brew and explains _why_ installing manually is not recommended not sure how much cleared I can be.

I don’t provide nor condone any of these so called “guides”, the only third-party I condone is PortingKit (where I’m a member).

**Comment by ebow77** [comment]
I guess I wish the wiki home page or [Wineskin-Winery page](https://github.com/Gcenx/WineskinServer/wiki/Wineskin-Winery) mentioned using brew. Instead the Winery page says to download the latest release archive. I guess I could have kept scrolling the homepage and found the readme, but I notice the wiki link first so that's where I went. 

Also I saw  the release name on the right side of the project homepage: "V1.8.4: Wineskin-Winery-1.8.4" and figured I should dig into the Winery folder. 

Maybe it's me, I rarely play try out projects like this. But I'm [not the only one](https://www.reddit.com/r/wineskin/comments/kg128z/que_wineskin_on_mac_os_sierra/) who didn't get it right.

**Reply by Gcenx** [reply]
While the wiki can be updated the README.md still contains this information and is almost _always_ ignored.
https://github.com/Gcenx/WineskinServer#how-to-install
https://github.com/Gcenx/WineskinServer/blob/3a9caf33803437a09b09f1e9580470b25ccd980c/README.md#L7-L18

**Comment by noveleven** [comment]
I solved it, check the **~/Library/Application Support/Wineskin folder**, put the wrapper and engine in the relevant folders. The problem may be that the wrapper and engine cannot be downloaded due to local network reasons.
Or execute **sudo echo "185.199.109.133 raw.githubusercontent.com" >> /etc/hosts** in your terminal to f**k it done. Hope it helps someone in the future.
